test/solidity/Helpers/SwapperV2.t.sol

```diff
@@ -6,6 +6,34 @@ import { TestToken as ERC20 } from "../utils/TestToken.sol";
 import { LibAllowList, LibSwap, TestBase } from "../utils/TestBase.sol";
 import { SwapperV2 } from "lifi/Helpers/SwapperV2.sol";
 
+// Test AMM that simulates returning tokens due to better pricing
+contract TestPartialAMM {
```

Comment:
We already have a testAMM, please add the function there as it may be reused by other tests later on